docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -64,10 +62,10 @@ User need to set Elasticsearch settings before deploying the manifest:
 [float]
 ===== Running {agent} on master nodes
 
-Kubernetes master nodes can use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
+Kubernetes master nodes use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
 to limit the workloads that can run on them. The manifest for standalone {agent} defines
-tolerations so as to run on master nodes too and being able to collect metrics from the control plane
-components of Kuberentes (scheduler, controller manager)
+tolerations to run on master nodes, which collects metrics from the control plane
+components of Kuberentes (scheduler, controller manager).
```

Comment:
I want to make sure this is rephrased correctly!

Reply:
Looks good